interface1.html

```diff
@@ -536,6 +543,12 @@ <h2>Inputs
 			<p>String of terminals: <input type="text" name="inputToGen"> <button id="goButton" type="button">Build syntax</button><button class="orange-button" type="submit" style='margin-left: 5px'>Run GEN</button></p>
 
             <div id="treeUI" style="display: none">
+								<div class="constraint-row">
+									<span><input type="checkbox" name="genOptions" id="trimTrees">Trim Syntactic tree</span>
```

Comment:
Let's change this text to say "Prune phonologically-contentless syntactic nodes"